### PR TITLE
Common models for Status, Authority and Taxpayer

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/models/common_models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from data_ocean.models.main import DataOceanModel
+
+class Status(DataOceanModel):
+    name = models.CharField(max_length=100, unique=True)
+
+class Authority(DataOceanModel):
+    name = models.CharField(max_length=500, unique=True)
+    code = models.CharField(max_length=10, unique=True, null=True)
```
Comment (criticus) [inline data_ocean/models/common_models.py:9]: what is code? are their predefined somewhere?

Comment (Iva-khar) [reply data_ocean/models/common_models.py:9]: fo sample:
```
AUTHORITY_NAME                  ДПI У ЦЕНТРАЛЬНОМУ РАЙОНI М.ДНIПРА ГУ ДФС У ДНIПРОПЕТРОВСЬКIЙ ОБЛАСТI
                 AUTHORITY_CODE                  39533006
```

Comment (IegorSoboliev) [reply data_ocean/models/common_models.py:9]: As far as I can see code here is a EDRPO of state body

Comment (IegorSoboliev) [reply data_ocean/models/common_models.py:9]: An example https://dtkt.com.ua/show/0sid0329.html

Comment (IegorSoboliev) [reply data_ocean/models/common_models.py:9]: All such data should be there http://lg.ukrstat.gov.ua/inform_z_edrpou.php.htm

File: data_ocean/models/common_models.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class TaxpayerType(DataOceanModel):
+    name = models.CharField(max_length=200, unique=True)
```
Comment (Iva-khar) [inline data_ocean/models/common_models.py:12]: ABC order )

**Pull request merged**